atc/db/pipeline.go

```diff
@@ -653,6 +654,40 @@ func (p *pipeline) Rename(name string) error {
 	return err
 }
 
+func (p *pipeline) Move(destinationTeamName string) error {
+	destinationTeamID, err := psql.Select("id").
+		From("teams").
+		Where(sq.Eq{
+			"name": destinationTeamName,
+		}).
+		RunWith(p.conn).Query()
+
+	_, err = psql.Update("pipelines").
+		Set("team_id", destinationTeamID).
+		Where(sq.Eq{
+			"id": p.id,
+		}).
+		RunWith(p.conn).
+		Exec()
+
+	if err != nil {
+		return err
+	}
+
+	_, err = psql.Update("builds").
+		Set("team_id", destinationTeamID).
+		Where(sq.Eq{
+			"id": p.id,
```

Comment:
This should be `"pipeline_id"` instead of `"id"`. The `"id"` represents a build id because you are updating the builds table.

Reply:
Updated.

Reply:
Oh I actually meant the left side of this `sq.Eq`, it should be `"pipeline_id"` rather than `"id"` because right now it's going to only update one row from the builds table that has a build `"id"` of your `p.id` (pipeline id) value.